A-encoding-base64.md

```diff
@@ -55,7 +55,7 @@ var decodedString = string(decoded)
 fmt.Println(decodedString)
 ```
 
-Fungsi `base64.StdEncoding.EncodedLen(len(data))` menghasilkan informasi lebar data-ketika-sudah-di-encode. Nilai tersebut kemudian ditentukan sebagai lebar alokasi tipe `[]byte` pada variabel `encoded` yang nantinya digunakan untuk menampung hasil encoding.
+Fungsi `base64.StdEncoding.EncodedLen(len(data))` menghasilkan informasi lebar dari variable `data` ketika sudah di-encode. Nilai tersebut kemudian ditentukan sebagai lebar alokasi tipe `[]byte` pada variabel `encoded` yang nantinya digunakan untuk menampung hasil encoding.
```

Comment:
And for this one, I think it sounds better with `... lebar variabel data ...`. so can you remote the `dari` word?